README.md

```diff
@@ -31,15 +32,10 @@ Call `katex.render` with a TeX expression and a DOM element to render into:
 katex.render("c = \\pm\\sqrt{a^2 + b^2}", element);
 ```
 
-To avoid escaping the backslash (double backslash), you can use
-[`String.raw`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/raw)
-(but beware that `${`, `\u` and `\x` may still need escaping):
-```js
-katex.render(String.raw`c = \pm\sqrt{a^2 + b^2}`, element);
-```
-
 If KaTeX can't parse the expression, it throws a `katex.ParseError` error.
```

Comment:
Is it worth saying "see https://khan.github.io/KaTeX/docs/api.html for details"? (to cover the removed part)

Reply:
@edemaine I don't think `String.raw` is as important as others.